src/l500/l500-color.cpp

```diff
@@ -31,6 +31,7 @@ namespace librealsense
         color_ep->register_pu(RS2_OPTION_HUE);
         color_ep->register_pu(RS2_OPTION_SATURATION);
         color_ep->register_pu(RS2_OPTION_SHARPNESS);
+        color_ep->register_pu(RS2_OPTION_AUTO_EXPOSURE_PRIORITY);
```

Comment:
Please verify that there is no (FW_VER>=TBD)  constrain for those options, and adjust accordingly.
Additional check:
- `rs-enumerate-devices -o`
- `Realsense-viewer`
- Live unit-tests